images/python/Alpine/apache.conf

```diff
@@ -0,0 +1,23 @@
+LoadModule wsgi_module "/usr/local/lib/python3.9/site-packages/mod_wsgi/server/mod_wsgi-py39.cpython-39-x86_64-linux-gnu.so"
+ServerName localhost
```

Comment:
May need to set errorlog and transferlog to stdout

https://serverfault.com/questions/711168/writing-apache2-logs-to-stdout-stderr

Reply:
Wait, maybe that's not necessary if supervisord.conf is configured right